assets/src/components/amp-support/error-sources.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * To render error source information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Error data.
+ * @return {JSX.Element|null} HTML markup for error source data.
+ */
+export function ErrorSources( { data: errorSources } ) {
```

Comment:
`ErrorSources` and `Errors` components are almost identical. How about using just 1 component for both use cases. We only need to expose the title and the description as props, i.e.:

```js
export function Errors( { title, description } ) {
	return (
		<details open={ false }>
			<summary>
				{ `${ title }` }
			</summary>
			<div className="detail-body">
				<p>
					{ description }
				</p>
			</div>
		</details>
	);
}
```

And then use it like:

```js
{ Array.isArray( data.errorSources ) && (
	<Errors
		title={ sprintf(
			/* translators: Placeholder is the number of error sources */
			__( 'Error Sources (%d)', 'amp' ),
			data.errorSources.length,
		) }
		description={ __( 'Please check "Raw Data" for all error source information.', 'amp' ) }
	/>
) }
```

Reply:
@delawski I have updated it. However, instead of what you suggested I introduced a new component. and used in both Errors and ErrorSource components. Please have a look.

Reply:
That makes sense. I have removed `Errors` and `ErrorSources` and used the `Details` component in `AMPSupport` itself.